dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionChoiceBehaviorConfigurationContext.cs

```diff
@@ -10,6 +11,20 @@ namespace Microsoft.SemanticKernel;
 [Experimental("SKEXP0001")]
 public sealed class FunctionChoiceBehaviorConfigurationContext
```

Comment:
Can the caller instantiate a `Context` ? As far as I understand, the context should be `internal` and provided as a parameter to a delegate the caller can modify, similar to how `filters` work.

```suggestion
internal sealed class FunctionChoiceBehaviorConfigurationContext
```

Reply:
This context lives in the abstracts project, but it's instantiated in AI connector projects. So, unfortunately, it can't be internal unless we use the InternalsVisible attribute.